### DISCUSSION TITLE
Problem with  Dracula for MySQL Workbench on debian 12.1

### DISCUSSION THREAD

**Discussion by abdrrahim2002** [dracula/dracula-theme#897] [Q&A] [answered]
I install mysql-workbench-community for my dabian 12.1 using 'snap' method but when i try to edit  the 'code_editor.xml' i can't because it's 'snap' and i don't know how to edit a 'snap' package .

i install the snap version  because the 'deb' version is for ubuntu and when i try to install it and error appear so i have to install it using 'snap' method.

can you help me i hate that white theme.

**Comment by luxonauta** [comment, marked as answer]
Thanks for providing this feedback. ✨

But there is a better place to share the same, which is the project repository, where you can talk to the theme maintainers.
